### PR TITLE
[Project Submission] filswan by NBFS Canada

### PR DESCRIPTION
**Pull request by flyworker**

### CODE REVIEW

File: projects/application/filswan.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+![Filswan](https://github.com/nebulaai/swan/blob/main/doc/swan.png)
+
+## Category 
+application
+
+## Project Description
+Swan is a marketplace for Filecoin miners, clients post/bidding deals the online.
+
+The platform also provides powerful deal lifecycle management, fast file retrieval, Petabyte-Scale data migration, content network delivery, multi-data center caching, encryption, micro payment and edge computing services. Miners on swan platform can provide additional services according to their infrastructure and capabilities. 
+
```
Comment (jennijuju) [inline projects/application/filswan.md:11]: Could you please add 

```
## Project Status
<!--brainstorming, fundraising, under development, beta, shipped, etc-->
```

File: projects/application/filswan.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+Baoyuan Sun
+
+Yi Xie
+
+Mike Li
+
+
+## How the community can engage
+
```
Comment (jennijuju) [inline projects/application/filswan.md:67]: To join the show case, could you please create a discussion in this repo too?

**Review by jennijuju — CHANGES_REQUESTED** [review]

**Comment by jennijuju** [comment]
Thank you for submitting and do you want your future updates to be featured in Filecoin newsletter?

File: projects/application/filswan.md (hunk 2 above)
Comment (flyworker) [reply projects/application/filswan.md:69]: addhttps://github.com/filecoin-project/community/discussions/69

**Review by flyworker — COMMENTED** [review]
> Thank you for submitting and do you want your future updates to be featured in Filecoin newsletter?

yes

**Review by jennijuju — APPROVED** [review]

**Pull request merged**